### ISSUE TITLE
Crash Report Error code: STATUS_BREAKPOINT

### ISSUE THREAD

**Issue by Midz10** [brave/brave-browser#17011]
IMPORTANT: Your crash has already been automatically reported to our crash system. Please file this bug only if you can provide more information about it.

Brave Version: 1.28.83  Chromium: 92.0.4515.93
Operating System: Windows NT 10.0.22000

URL (if applicable) where crash occurred:

https://mail.yahoo.com/d/folders/1

https://www.twitch.tv/directory/following

Can you reproduce this crash?

Opening most sites will make it happen regularly.
 
What steps will reproduce this crash? (If it's not reproducible, what were you doing just before the crash?)
1. Opening any site.
2.
3.

Updated Windows, GPU drivers, Brave and tried Brave beta. This started to happen after the update before the last one. 

****DO NOT CHANGE BELOW THIS LINE****
Crash ID: crash/a4020f00-ad33-a506-0000-000000000000

**Comment by Midz10** [comment]
Im pretty sure I sent over 500 reports since the issue started. And since their is no fix or even a response I'm done with this browser. Switching to something else.  Even opening this page the tab crashed 3 times.

**Comment by bsclifton** [comment]
Sorry to hear that @Midz10 - I'm checking out the crash report now. It's failing when initializing the v8 snapshot

Any ideas @iefremov or @bridiver? Crash report is here:
https://brave.sp.backtrace.io/p/brave/debug?time=quarter&filters=(_rxid%3Da4020f00-ad33-a506-0000-000000000000)&fingerprint=9b9e1c6d25e1e22b72b26e9f74314faefac27a0649ae0dc07a2eb870a38af2b0&debug=(%2221a1056%22,0,0)

**Comment by iefremov** [comment]
by any chance, do you have anything on brave://flags enabled, @Midz10 ?

**Comment by iefremov** [comment]
the stack
```
[ 00 ] v8::internal::Snapshot::Initialize(v8::internal::Isolate *)
[ 01 ] operator new(unsigned __int64)
[ 02 ] _tailMerge_esent.dll
[ 03 ] _tailMerge_esent.dll
[ 04 ] v8::Isolate::Initialize(v8::Isolate *,v8::Isolate::CreateParams const &)
[ 05 ] v8::ResourceConstraints::ConfigureDefaults(unsigned __int64,unsigned __int64)
[ 06 ] gin::IsolateHolder::IsolateHolder(scoped_refptr<base::SingleThreadTaskRunner>,gin::IsolateHolder::AccessMode,gin::IsolateHolder::AllowAtomicsWaitMode,gin::IsolateHolder::IsolateType,gin::IsolateHolder::IsolateCreationMode)
[ 07 ] _tailMerge_esent.dll
[ 08 ] _tailMerge_esent.dll
[ 09 ] _tailMerge_esent.dll
[ 10 ] blink::V8PerIsolateData::Initialize(scoped_refptr<base::SingleThreadTaskRunner>,blink::V8PerIsolateData::V8ContextSnapshotMode)
[ 11 ] _tailMerge_esent.dll
[ 12 ] blink::WorkerBackingThread::InitializeOnBackingThread(blink::WorkerBackingThreadStartupData const &)
[ 13 ] base::`anonymous namespace'::QPCNow()
[ 14 ] _tailMerge_esent.dll
[ 15 ] ThreadProfiler::StartOnChildThread(metrics::CallStackProfileParams::Thread)
[ 16 ] base::sequence_manager::internal::WorkQueueSets::OnPopMinQueueInSet(base::sequence_manager::internal::WorkQueue *)
[ 17 ] _tailMerge_esent.dll
[ 18 ] void blink::WorkerThread::InitializeOnWorkerThread(class std::__1::unique_ptr<blink::GlobalScopeCreationParams,std::__1::default_delete<blink::GlobalScopeCreationParams> >, const class absl::optional<blink::WorkerBackingThreadStartupData> & const, class std::__1::unique_ptr<blink::WorkerDevToolsParams,std::__1::default_delete<blink::WorkerDevToolsParams> >)
[ 19 ] base::sequence_manager::internal::TaskQueueImpl::NotifyWillProcessTask(base::sequence_manager::Task const &,bool)
[ 20 ] _tailMerge_esent.dll
[ 21 ] base::sequence_manager::internal::SequenceManagerImpl::SelectNextTask(base::sequence_manager::internal::SequencedTaskSource::SelectTaskOption)
[ 22 ] void base::debug::GlobalActivityTracker::ScopedThreadActivity::ScopedThreadActivity(const void *, const void *, base::debug::Activity::Type, const union base::debug::ActivityData & const, bool)
[ 23 ] _tailMerge_esent.dll
[ 24 ] _tailMerge_esent.dll
[ 25 ] _tailMerge_esent.dll
[ 26 ] blink::WorkerThread::Start(std::__1::unique_ptr<blink::GlobalScopeCreationParams,std::__1::default_delete<blink::GlobalScopeCreationParams> >,absl::optional<blink::WorkerBackingThreadStartupData> const &,std::__1::unique_ptr<blink::WorkerDevToolsParams,std::__1::default_delete<blink::WorkerDevToolsParams> >)
[ 27 ] _tailMerge_esent.dll
[ 28 ] static void base::internal::Invoker<base::internal::BindState<void (blink::WorkerThread::*)(std::__1::unique_ptr<blink::GlobalScopeCreationParams,std::__1::default_delete<blink::GlobalScopeCreationParams> >, const absl::optional<blink::WorkerBackingThreadStartupData> &, std::__1::unique_ptr<blink::WorkerDevToolsParams,std::__1::default_delete<blink::WorkerDevToolsParams> >),WTF::CrossThreadUnretainedWrapper<blink::WorkerThread>,std::__1::unique_ptr<blink::GlobalScopeCreationParams,std::__1::default_delete<blink::GlobalScopeCreationParams> >,absl::optional<blink::WorkerBackingThreadStartupData>,std::__1::unique_ptr<blink::WorkerDevToolsParams,std::__1::default_delete<blink::WorkerDevToolsParams> > >,void ()>::RunOnce(class base::internal::BindStateBase *)
[ 29 ] base::ThreadLocalStorage::Slot::Get()
[ 30 ] base::TaskAnnotator::RunTask(char const *,base::PendingTask *)
[ 31 ] mojo::SimpleWatcher::ArmOrNotify()
[ 32 ] base::ThreadLocalStorage::Slot::Get()
[ 33 ] base::WatchHangsInScope::WatchHangsInScope(base::TimeDelta)
[ 34 ] blink::WorkerThread::Start(std::__1::unique_ptr<blink::GlobalScopeCreationParams,std::__1::default_delete<blink::GlobalScopeCreationParams> >,absl::optional<blink::WorkerBackingThreadStartupData> const &,std::__1::unique_ptr<blink::WorkerDevToolsParams,std::__1::default_delete<blink::WorkerDevToolsParams> >)
[ 35 ] mojo::SimpleWatcher::ArmOrNotify()
[ 36 ] static void content::`anonymous namespace'::CreateEmbeddedWorker(class scoped_refptr<base::SingleThreadTaskRunner>, class base::WeakPtr<content::RenderThreadImpl>, class mojo::PendingReceiver<blink::mojom::EmbeddedWorkerInstanceClient>)
[ 37 ] mojo::internal::GenericCallbackBinderWithContext<void>::BindInterface(mojo::ScopedHandleBase<mojo::MessagePipeHandle>)
[ 38 ] _tailMerge_esent.dll
[ 39 ] base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWork()
[ 40 ] base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::OnEndWorkItem()
[ 41 ] _tailMerge_esent.dll
[ 42 ] base::MessagePumpDefault::Run(base::MessagePump::Delegate *)
[ 43 ] operator new(unsigned __int64)
[ 44 ] static void * base::internal::PartitionMalloc(const struct base::allocator::AllocatorDispatch *, unsigned __int64, void *)
[ 45 ] base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::Run(bool,base::TimeDelta)
[ 46 ] base::RunLoop::Run(base::Location const &)
[ 47 ] blink::scheduler::WorkerThread::SimpleThreadImpl::Run()
[ 48 ] _tailMerge_esent.dll
[ 49 ] _tailMerge_esent.dll
[ 50 ] blink::scheduler::WorkerThread::SimpleThreadImpl::Run()
[ 51 ] static unsigned long base::`anonymous namespace'::ThreadFunc(void *)
[ 52 ] 0x7ffc489554e0
[ 53 ] 0x7ffc496e485b
[ 54 ] TerminateProcessOnMemoryExhaustion
```

**Comment by iefremov** [comment]
@Midz10 could you please also check memory pressure? I.e. take a screenshot of 
hamburger menu -> More tools -> Task Manager

and your windows task manager?
it could be an out of memory problem

**Comment by Midz10** [comment]
> by any chance, do you have anything on brave://flags enabled, @Midz10 ?

I didn't change anything there. I deleted and installed the browser multiple times. Now running the beta version and the same issue is still there.

**Comment by Midz10** [comment]
By mistake.

**Comment by bsclifton** [comment]
@Midz10 uninstalling browser won't clear your profile; you can try a new profile by moving your old one out of the way. On Windows, the profile would be under `C:\Users\<USERNAME>\AppData\local\BraveSoftware`. You can rename the `Brave-Browser-Beta` folder as `Brave-Browser-Beta2` or something like this

**Comment by Midz10** [comment]
> @Midz10 could you please also check memory pressure? I.e. take a screenshot of
> hamburger menu -> More tools -> Task Manager
> 
> and your windows task manager?
> it could be an out of memory problem

Will take screenshots when I get home.  I have 32gb ram. I tried to reset BIOS and my overclock and XMP profile and used default profile and it didn't fix the issue.

**Comment by Midz10** [comment]
> by any chance, do you have anything on brave://flags enabled, @Midz10 ?

Every time I open brave://flags the tab crashes.

**Comment by Midz10** [comment]
> @Midz10 could you please also check memory pressure? I.e. take a screenshot of
> hamburger menu -> More tools -> Task Manager
> 
> and your windows task manager?
> it could be an out of memory problem

![Screenshot 2021-08-14 025429](https://user-images.githubusercontent.com/87559610/129427935-6a7dbaf4-ec15-4bb4-8224-d4018a428421.png)
![Screenshot 2021-08-14 025336](https://user-images.githubusercontent.com/87559610/129427936-51891abe-b035-4ad4-bbd2-eef8e2a3f2e6.png)

**Comment by Midz10** [comment]
> Sorry to hear that @Midz10 - I'm checking out the crash report now. It's failing when initializing the v8 snapshot
> 
> Any ideas @iefremov or @bridiver? Crash report is here:
> https://brave.sp.backtrace.io/p/brave/debug?time=quarter&filters=(_rxid%3Da4020f00-ad33-a506-0000-000000000000)&fingerprint=9b9e1c6d25e1e22b72b26e9f74314faefac27a0649ae0dc07a2eb870a38af2b0&debug=(%2221a1056%22,0,0)
> 
> This might be Chromium 92 related. Do you know the last version which worked @Midz10?

I had this issue since the last two-three updates. Where can I find a Brave version before the Chromium 92? Tried Chrome and had the same issue. So Chromium probably is the issue.

**Comment by bsclifton** [comment]
@Midz10 you can find the Chromium 91 version here:
https://github.com/brave/brave-browser/releases/tag/v1.26.77

Specifically, you'll want to download this file:
https://github.com/brave/brave-browser/releases/download/v1.26.77/BraveBrowserStandaloneSetup.exe

You'll have to uninstall your current version of Brave, as this is newer. You won't lose your profile! so no worries there. But I would make a backup of the profile directory (or move it out of the way for testing).  After you remove the newer version, you can install this. If you visit the about Brave page, it will update unfortunately. But as long as you don't restart you'll still be on Chromium 91

You can try this and see if the problem happens. Let us know!

**Comment by iefremov** [comment]
Just in case - newer profile (from c92) could be incompatible with an older browser version (aka c91). So make sure you backup your data.

**Comment by Midz10** [comment]
I tried Chromium 91 version. Now getting another code (status_access_violation) but the same thing is happening. almost every other tab will crash. Tried fresh install with no profile (removed it from appdata before installing) and the welcome tab crashed :-) . 

I lost hope.  I installed Firefox on this PC. My other PC and laptop have Brave latest with sync on (everything synced)  and not a single crash. 

On this PC I upgraded from win10 to 11, reset BIOS, removed overclock profile and tried default, checked memory with "Windows Memory Diagnostic" no  issue found.
